Formula/coccinelle.rb

```diff
@@ -3,7 +3,7 @@ class Coccinelle < Formula
   homepage "http://coccinelle.lip6.fr/"
   url "http://coccinelle.lip6.fr/distrib/coccinelle-1.0.6.tgz"
   sha256 "8452ed265c209dae99cbb33b67bc7912e72f8bca1e24f33f1a88ba3d7985e909"
-  revision 1
+  revision 2
```

Comment:
this is a small enough change not to warrant a revision bump because the existing installations aren't actually broken